src/validation/admin/adminLogin/loginValidation.ts

```diff
@@ -0,0 +1,10 @@
+
+export const validateEmail = (email: string): boolean => {
+    const emailRegex = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6}$/;
```

Comment:
move this regex patter to config > regex

Reply:
Fixed i am raised new commit in same branch